DotNet/emitter.js

```diff
@@ -534,7 +560,7 @@
 function setUpBrowsers(test, addHook) {
 	let headless = ("env" in test) && ("headless" in test.env) && (test.env.headless === false) ? false : true
 	let legacy = ("env" in test) && ("legacy" in test.env) && (test.env.legacy === true) ? true : false
-	let css = ("stitchMode" in test.config) && (test.config.stitchMode.toUpperCase().localeCompare('CSS')) ? false : true // localeCompare returns 0 when the strings are equal
+	let css = ("stitchMode" in test.config) && (test.config.stitchMode.toUpperCase().localeCompare('SCROLL')) ? true : false // localeCompare returns 0 when the strings are equal
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
	let css = ("stitchMode" in test.config) && (test.config.stitchMode.toUpperCase().localeCompare('SCROLL')) ? true : false; // localeCompare returns 0 when the strings are equal
```